cmd/diagnostic/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM docker:17.12-rc-dind
```

Comment:
I was also thinking why we needed the `dind` for this tool, but this is so that we can use it for older daemons, which do not have this functionality built-in, correct?

Should we have two versions of the image? one "minimal" (just the binary), and one `dind`?